src/utilities/common/incrementalResult.ts

```diff
@@ -28,6 +30,18 @@ export function isExecutionPatchResult<T>(
   );
 }
 
+// This function detects an Apollo payload result before it is transformed
+// into a FetchResult via HttpLink; it cannot detect an ApolloPayloadResult
+// once it leaves the link chain.
+export function isApolloPayloadResult<T>(
```

Comment:
```suggestion
export function isApolloPayloadResult(
```
Looks like `T` is no longer used/needed.

Reply:
https://github.com/apollographql/apollo-client/pull/10592/commits/e2a727fa27638a3c7af0f58d8093b5ed9986c987